test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -823,6 +823,12 @@ Verb-Noun -Param1 Hello ^
         $res.CompletionMatches[0].CompletionText | Should -Be "Get-ChildItem"
     }
 
+    it 'Should prefer the default parameterset when completing positional parameters' {
+        $ScriptInput = 'Get-ChildItem | Where-Object '
+        $res = TabExpansion2 -inputScript $ScriptInput -cursorColumn $ScriptInput.Length
+        $res.CompletionMatches[0].CompletionText | Should -Be "Attributes"
+    }
```

Comment:
The test works for me without the change.

Reply:
That's because the change that broke it hasn't made its way to a real release yet. If you try it on a daily build it won't work.